### PR TITLE
Feature/1071 content submission mail notification

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -373,6 +374,36 @@ public function p4_save_page_type( $post_id ) {
...
+		if( $emails ) {
+			$headers  = 'MIME-Version: 1.0' . "\r\n";
+			$headers .= 'Content-type: text/html; charset=iso-8859-1' . "\r\n";
+			$name     = $post->post_name;
+			$subject  = __( 'New post published', 'planet4-master-theme' );
+			$body     = sprintf( 'Hello Team, <br><br>' );
+			$body    .= sprintf( 'The post <strong> %s </strong> has been published. <br>', $name );
+			$body    .= sprintf( 'Post link : %s <br><br>', get_permalink( $post ) );
+			$body    .= sprintf( 'Thanks<br>' );
+			$body    .= sprintf( 'Greenpeace IT' );
```
Comment (angtheod) [inline functions.php]: @sagarsdeshmukh Should these strings be translatable?

**Comment by angtheod** [comment]
@koyan On Jira ticket description it mentions that web editors should be notified of new content based on their notification preferences. How do we apply that?

**Comment by angtheod** [comment]
@sagarsdeshmukh I'd suggest that we start separating concerns included in the P4_Master_Site class.
It is getting bigger and bigger and it will be harder to maintain later on.
We could add separate functionality in a new P4_<name> class and add it as a service inside the array that the P4_Master_Site class takes as an argument. What do you think?

File: functions.php (hunk 1 above)
Comment (sagarsdeshmukh) [reply functions.php]: Yes it is but I assume its just an initial basic version of mail content and in future, we will change this mail template  @koyan : Do you have any suggestion for this ?

**Comment by sagarsdeshmukh** [comment]
Hi @angtheod I do agree and I will update accordingly.

**Comment by koyan** [comment]
> @koyan On Jira ticket description it mentions that web editors should be notified of new content 
> based on their notification preferences. How do we apply that?
Hmm, I missed that. Currently there are no notification settings for wp admin users. 
We should ask the product owner if she wants to extend this to creating notification settings, or if for now all "Editors" should receive notifications.

File: functions.php (hunk 1 above)
Comment (27Bslash6) [reply functions.php]: And should this test be in UTF instead of ISO8859?

**Pull request closed without merging**